scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -527,7 +540,7 @@ class Router(formatOps: FormatOps) {
         )
       // Opening [ with no leading space.
       // Opening ( with no leading space.
-      case FormatToken(
+      case tok @ FormatToken(
```

Comment:
i wouldn't add tok. you can always refer to the actual variable, formatToken.

Reply:
Fixed!